addons/website/models/website_menu.py

```diff
@@ -26,6 +27,8 @@ def _default_sequence(self):
     child_id = fields.One2many('website.menu', 'parent_id', string='Child Menus')
     parent_path = fields.Char(index=True)
     is_visible = fields.Boolean(compute='_compute_visible', string='Is Visible')
+    mega_menu_content = fields.Html('Mega Menu Content', translate=html_translate, default=False)
+    mega_menu_classes = fields.Char('Mega Menu Classes')
```

Comment:
useless labels, those are the default

Reply:
I'm not resolving without adapting the code. So, to be clear, should I use html_translate on mega_menu_content or not ? + I remove 'Mega Menu Content' and 'Mega Menu Classes'

Reply:
The conversation was resolved and the labels were still there aren't they... ? :anguished: 
Yes and yes.